packages/react-native/React/Fabric/Mounting/ComponentViews/ScrollView/RCTScrollViewComponentView.mm

```diff
@@ -203,21 +204,24 @@ - (void)_keyboardWillChangeFrame:(NSNotification *)notification
                                                from:self
                                            forEvent:nil]) {
     if (CGRectEqualToRect(_firstResponderFocus, CGRectNull)) {
-      // Text input view is outside of the scroll view.
-      return;
-    }
-
-    CGRect viewIntersection = CGRectIntersection(self.firstResponderFocus, keyboardEndFrame);
+      UIView *inputAccessoryView = _firstResponderViewOutsideScrollView.inputAccessoryView;
+      if (inputAccessoryView) {
+        // Text input view is within the inputAccessoryView.
+        contentDiff = keyboardEndFrame.origin.y - keyboardBeginFrame.origin.y;
+      }
```

Comment:
Do we still want an early return here? Or should we have a general check for `contentDiff == 0`?

Reply:
I prefer to maintain the same logic as the old architecture, which does not perform early return checks or check for `contentDiff == 0`.